protos/peers.proto

```diff
@@ -89,6 +89,16 @@ message S3Config {
   PostgresConfig metadata_db = 7;
 }
 
+message ClickhouseConfig{
+  string host = 1;
+  uint32 port = 2;
+  string user = 3;
+  string password = 4;
+  string database = 5;
+  optional string metadata_schema = 6;
```

Comment:
would appreciate comments to tell what the last 2 fields specify

Reply:
sure, i will remove metadata_schema field, its not needed